app/app.py

```diff
@@ -0,0 +1,32 @@
+import gspread
+from google.oauth2.service_account import Credentials
+from dotenv import load_dotenv
+import json
+import os
+import time
+import redis
+# Define the scope of the application
+scope = ["https://www.googleapis.com/auth/spreadsheets"]
```

Comment:
I think we can refine the scope of this to readonly in this line (the actual service account should definitely be configured as readonly anyway in the time being).